windows/src/buildtools/Makefile

```diff
@@ -4,8 +4,8 @@
 
 NOTARGET_SIGNCODE=yes
 
-TARGETS=devtools sentrytool tds2dbg genkmnp getfilelocks isadmin mkver \
-  xslt buildunidata build_standards_data buildpkg test-klog delphiprojectmanager #help
```

Comment:
Since it was kind of glossed over in the description, I assume that 

> Remove some obsolete projects along the way.

refers to these changes?  So most of those "obsolete projects" were "build tools"?

I did a detailed-enough inspection to notice that `genkmnp`, the first removed entry here, is wholesale deleted by this PR.

Reply:
Yes, sorry, should have itemised them. The obsolete projects were build tools that are no longer referenced in the build (mostly due to improved build environments):

* genkmnp - generates a keyboard that contains every virtual key, but very obsolete, not used, better to rewrite if we need
* getfilelocks - check for locked files before build, no longer required
* isadmin - test if running as administrator, no longer used
* xslt - xslt transform tool, no longer used